File: Baekjoon/4단계 1차원 배열/공 넣기.py
n, m = map(int, input().split())
result = [0] * n
for _ in range(m):
    i, j, k = map(int, input().split())
    for l in range(i - 1, j):
        result[l] = k
result = map(str, result)
print(' '.join(result))

File: Baekjoon/js/4단계/공 넣기.js
const fs = require('fs');

const input = fs.readFileSync('/dev/stdin').toString().split('\n');

const [numbers, inputLength] = input[0].split(' ').map(Number);

const ballNumbers = new Array(numbers);
ballNumbers.fill(0);

for (let i = 1; i <= inputLength; i += 1) {
  const [basket1, basket2, ball] = input[i].split(' ').map(Number);
  for (let j = basket1 - 1; j <= basket2 - 1; j += 1) {
    ballNumbers[j] = ball;
  }
}

console.log(ballNumbers.join(' '));
